src/data/seq/seq.lean

```diff
@@ -49,6 +49,15 @@ end⟩
 /-- Get the nth element of a sequence (if it exists) -/
 def nth : seq α → ℕ → option α := subtype.val
 
+@[simp] theorem nth_mk (f hf) : @nth α ⟨f, hf⟩ = f := rfl
+
+@[simp] theorem nth_nil (n : ℕ) : (@nil α).nth n = none := rfl
+@[simp] theorem nth_cons_zero (a : α) (s : seq α) : (cons a s).nth 0 = a := rfl
```

Comment:
```suggestion
@[simp] theorem nth_cons_zero (a : α) (s : seq α) : (cons a s).nth 0 = some a := rfl
```
Way more lemmas about `some a` than `coe a`.

Reply:
Should `some` be the normal form?

Reply:
I don't think we get to choose due to reducibility problems with `with_top` etc